dev-cluster

```diff
@@ -185,6 +192,16 @@ if [[ ${_action} == "" ]]; then
 	exit 1
 fi
 
+if (( "$#" == 2 )); then
+  _flag=$2
+  if [[ ${_flag} != "--use-ovn" ]]; then
+  	echo "Error:
+  	Unsupported flag!!!
+  	"
+  fi
+  devcluster::export-ovn-as-network-type
+fi
+
```

Comment:
I think this should be handled inside `function devcluster::create-cluster` as this is relevant only to this function
We will might want to add different arguments to different functions and to handle those differently
So if you move it, also need to change Usage message to include this option only in `create-cluster` command